### PR TITLE
PE-2637: Print data to stdout

### PR DESCRIPTION
**Pull request by hanpeter**
## What does this PR do?
1. This PR adds a method to load any arbitrary custom reports to `krux-cloud-health`.
1. This PR also drastically changes `cloud-health-to-graphite` so that it prints out the data to stdout instead of using `statsd`. It also no longer gets data from Cost History report.

## Why is this change being made?
The new custom data is better for our usage. Sending data directly to graphite via netcat allows me to add data to the past.

## How was this tested? How can the reviewer verify your testing?
The change is tested on the development environment and via unit tests.

## Where should the reviewer start?
`bin/cloud_health_to_graphite.py`

## Completion checklist

- [x] The pull request has been appropriately labelled according to
  our [conventions](https://kruxdigital.jira.com/wiki/display/EN/Changes+and+Peer+Review)
- [x] The change has unit & integration tests as appropriate.

### CODE REVIEW

File: bin/cloud_health_to_graphite.py [hunk 1]
```diff
@@ -48,11 +49,16 @@ def add_cli_arguments(self, parser):
         group = get_group(parser, self.name)
 
         group.add_argument(
-            '--interval',
+            'report_id',
```
Comment (dcrampton-sfdc) [inline bin/cloud_health_to_graphite.py:58]: No leading hyphens for this?

Comment (hanpeter) [reply bin/cloud_health_to_graphite.py:58]: `report_id` is a required option.

**Review by dcrampton-sfdc — APPROVED** [review]

**Review by dcrampton-sfdc — APPROVED** [review]

File: bin/cloud_health_to_graphite.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+class Application(krux_cloud_health.cli.Application):
+    NAME = 'cloud-health-to-graphite'
+
+    def __init__(self, name=NAME):
+
+        # Call to the superclass to bootstrap.
+        super(Application, self).__init__(name=name)
+
+        # XXX: Empty space and period causes issues with graphite. Replace it with underscore.
+        self.report_name = self.args.report_name.replace(' ', '_').replace('.', '_')
```
Comment (paul-krohn) [inline bin/cloud_health_to_graphite.py]: you'll probably want `[ \.]+` as the match expression?

**Review by paul-krohn — APPROVED** [review]
LGTM just had the 1 question.

**Review by paul-krohn — APPROVED** [review]
👍

**Pull request merged**